tests/e2e/export.go

```diff
@@ -103,17 +106,24 @@ func exportResource(h *create.Harness, obj *unstructured.Unstructured, expectati
 		}
 		exportURI = strings.ReplaceAll(exportURI, "{.spec.collection}", collection)
 	}
-
 	exportParams := h.ExportParams()
 	exportParams.IAMFormat = "partialpolicy"
 	exportParams.ResourceFormat = "krm"
 	outputDir := h.TempDir()
 	outputPath := filepath.Join(outputDir, "export.yaml")
 	exportParams.Output = outputPath
 	exportParams.URI = exportURI
-	if err := export.Execute(h.Ctx, &exportParams); err != nil {
-		h.Errorf("error from export.Execute: %v", err)
-		return ""
+	switch gvk.Kind {
+	case "SecretManagerSecretVersion":
+		// Skip run export.Execute because the SecretVersion servicemappings has a broken marker
```

Comment:
Can we only export if using direct?  Or just move to direct export everywhere?

Reply:
I guess in practice it's not very useful to export the SecretManagerSecretVersion, and potentially risky (exposing secret material)

Reply:
Agree. And agree. I hit an error around the scenario, and explored on what happened in the code. Lots of improvements are needed for Export.